src/main/java/com/nylas/HostedAuthentication.java

```diff
@@ -62,6 +62,7 @@ public AccessToken fetchToken(String authorizationCode) throws IOException, Requ
 		private String scopes = "";
 		private String loginHint = "";
 		private String state = "";
+		private Boolean forcePassword;
```

Comment:
may want to default this to false, null booleans in java are problematic

Reply:
Good call out!